txnprovider/txpool/pool_txn_parser.go

```diff
@@ -771,16 +772,20 @@ func parseTransactionBodyAA(payload []byte, p int, slot *TxnSlot) (int, error) {
 	return p, nil
 }
 
-func getAddress(payload []byte, p int, name string) (common.Address, int, error) {
+func getAddress(payload []byte, p int, name string) (*common.Address, int, error) {
 	dataPos, dataLen, err := rlp.ParseString(payload, p)
 	if err != nil {
-		return common.Address{}, 0, fmt.Errorf("%w: to len: %s", ErrParseTxn, err) //nolint
+		return nil, 0, fmt.Errorf("%w: to len: %s", ErrParseTxn, err) //nolint
```

Comment:
Forgot to remove this.

```suggestion
		return nil, 0, fmt.Errorf("%w: to len: %s", ErrParseTxn, err)
```